CMakeLists.txt

```diff
@@ -35,10 +35,12 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/modules/" ${CMAKE_MODULE_PATH})
 find_package(Magnum REQUIRED)
 
+include(CMakeDependentOption)
+
 # Parts of the library
 option(WITH_BULLET "Build BulletIntegration library" OFF)
 option(WITH_DART "Build DartIntegration library" OFF)
-option(WITH_EIGEN "Build EigenIntegration library" OFF)
+cmake_dependent_option(WITH_EIGEN "Build EigenIntegration library" ON "WITH_DART" OFF)
```

Comment:
Not needed, like I said there's just the header dependency ;)

Reply:
But if we haven't installed EigenIntegration, how are we going to find the header?

Reply:
The header is only needed when compiling the DartIntegration, and it's right in the directory next to DartIntegration :)

Reply:
Ahhh ok.. That's what you meant! OK. Doing this..